cmd/node/config/enableEpochs.toml

```diff
@@ -0,0 +1,70 @@
+[EnableEpochs]
```

Comment:
I think you can drop this "container". For example now you use something like `epochConfig.EnableEpochs.ScDeployEnableEpoch` while you could use `enableEpochsConfig.ScDeployEnableConfig`.
But also, the way it is right now can be useful if we want to add another section to this `toml` file. 
@iulianpascalau What do you think?

Reply:
I will leave the container as it is, as the `GasSchedule` section from `config.toml` will be moved as well